codility_training/lessons.lesson03.TimeComplexity.PermMissingElem/NowonLee.js

```diff
@@ -0,0 +1,17 @@
+function solution(A) {
+    // write your code in JavaScript (Node.js 8.9.4)
+    var r = 0
+    A.sort(function(a,b){return a-b})

```

Comment:
```suggestion
    A.srot((a, b) => a - b)
```
이렇게 Arrow Function을 이용해봅시다!